File: ReactNativeWithExpo/app/screens/welcomeScreen.js
import React from "react";
import { StyleSheet, View, Text, Image } from "react-native";

import kcolors from "../config/kcolors";
import kstyles from "../config/kstyles";
import AppButton from "../components/Button";
import Screen from "../components/Screen";

function WelcomeScreen({ navigation }) {
  return (
    <Screen style={kstyles.bgPrimary}>
      <Screen.Header>
        <Text style={[kstyles.text, styles.headTitle]}>
          Welcome to BTC Exchange
        </Text>
        <Text style={[kstyles.text, styles.headSubtitle]}>
          We are offering digital asset exchange with maximum security and
          advanced trading features.
        </Text>
      </Screen.Header>
      <Screen.Body>
        <View style={styles.middle}>
          <Image
            style={styles.logo}
            source={require("../assets/logo-center.png")}
          />
          <AppButton
            title="SIGN UP"
            type="primaryWhite"
            style={styles.btnSignup}
            onPress={() => navigation.navigate("Signup")}
          />
        </View>
      </Screen.Body>
      <Screen.Footer>
        <Text style={styles.textHasAccount}>Already a member?</Text>
        <AppButton
          title="Sign In"
          type="transparentWhite"
          onPress={() => navigation.navigate("Signin")}
        />
      </Screen.Footer>
    </Screen>
  );
}

const styles = StyleSheet.create({
  logo: {
    width: "100%",
    height: "80%",
    marginTop: 40,
  },
  headTitle: {
    color: kcolors.white,
    fontSize: 29,
    marginBottom: 20,
  },
  headSubtitle: {
    color: kcolors.secondary,
    textAlign: "center",
  },
  middle: {
    flex: 1,
    paddingLeft: 20,
    paddingRight: 20,
    width: "100%",
    alignItems: "center",
    justifyContent: "space-between",
  },
  textHasAccount: {
    color: kcolors.light,
  },
  btnSignup: {
    marginBottom: 15,
  },
});

export default WelcomeScreen;

File: flutter_app/lib/screens/welcomeScreen.dart
import 'package:flutter/material.dart';

import '../constants.dart';

class WelcomeScreen extends StatefulWidget {
  @override
  _WelcomeScreen createState() => _WelcomeScreen();
}

class _WelcomeScreen extends State<WelcomeScreen> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: null,
      body: Column(
        crossAxisAlignment: CrossAxisAlignment.stretch,
        mainAxisAlignment: MainAxisAlignment.start,
        children: <Widget>[
          Container(
            margin: EdgeInsets.only(top: 30.0),
            child: Column(
              children: <Widget>[
                SizedBox(
                  height: 50,
                ),
                Text(
                  'Welcome to BTC Exchange',
                  style: kTitleWhiteStyle,
                  textAlign: TextAlign.center,
                ),
                Padding(
                  padding: const EdgeInsets.all(30.0),
                  child: Text(
                    'We are offering digital asset exchange with maximum security and advanced trading features',
                    style: kNormalWhiteStyle,
                    textAlign: TextAlign.center,
                  ),
                ),
              ],
            ),
          ),
          Container(
            padding: EdgeInsets.only(top: 30.0, bottom: 30.0),
            child: Column(
              children: <Widget>[
                Image.asset('assets/images/logo-center.png'),
              ],
            ),
          ),
          Container(
            child: Column(
              children: <Widget>[
                Container(
                  margin: EdgeInsets.all(10.0),
                  child: RaisedButton(
                      shape: RoundedRectangleBorder(
                        borderRadius: BorderRadius.circular(24.0),
                        side: BorderSide(
                          color: Colors.white,
                        ),
                      ),
                      onPressed: () {},
                      color: Colors.white,
                      padding: EdgeInsets.all(12.0),
                      textColor: kPrimaryColor,
                      child: Container(
                        width: double.infinity,
                        child: Text(
                          'SIGN UP',
                          textAlign: TextAlign.center,
                          style: TextStyle(
                            fontSize: 16,
                          ),
                        ),
                      )),
                ),
                Text(
                  'Already a member?',
                  textAlign: TextAlign.center,
                  style: TextStyle(
                    color: kLightColor,
                    fontSize: 17.0,
                  ),
                ),
                Container(
                  margin: EdgeInsets.all(10.0),
                  child: RaisedButton(
                      shape: RoundedRectangleBorder(
                        borderRadius: BorderRadius.circular(24.0),
                        side: BorderSide(
                          color: kPrimaryColor,
                        ),
                      ),
                      onPressed: () {},
                      color: kPrimaryColor,
                      textColor: kWhiteColor,
                      padding: EdgeInsets.all(12.0),
                      child: Container(
                        width: double.infinity,
                        child: Text(
                          'Sign In',
                          textAlign: TextAlign.center,
                          style: TextStyle(
                            fontSize: 14,
                          ),
                        ),
                      )),
                ),
              ],
            ),
          ),
        ],
      ),
    );
  }
}
